libs/isograph-react/src/core/IsographEnvironment.ts

```diff
@@ -155,9 +162,8 @@ export function defaultMissingFieldHandler(
     const value = variables?.[variable];
 
     // TODO can we handle explicit nulls here too? Probably, after wrapping in objects
-    if (value != null) {
-      // @ts-expect-error
-      return { __link: value };
+    if (typeof value === 'string') {
+      return { __link: value, __typename: 'Pet' };
```

Comment:
Let's not have a `defaultMissingFieldHandler` if it requires hard coding `Pet` as a result of this 😂

See #211 for a proper solution. We can also hard code pet in the pet-demo missing field handler.